pkg/lib/config/authentication_flow.go

```diff
@@ -1441,12 +1441,16 @@ const (
 )
 
 type AuthenticationFlowAccountLinkingJSONPointer struct {
-	Pointer jsonpointer.T `json:"pointer,omitempty"`
+	Pointer string `json:"pointer,omitempty"`
```

Comment:
I don't quite get what this bug is about from the commit title. Can you elaborate?

Reply:
@louischan-oursky I found the problem on adding e2e test. Basically the problem is if I parse `/email` into a `jsonpointer.T`, then I serialize the config. It will become `#/email`.

Reply:
I see.